plugin/v2/plugin_linux.go

```diff
@@ -136,5 +139,32 @@ func (p *Plugin) InitSpec(execRoot string) (*specs.Spec, error) {
 		p.modifyRuntimeSpec(&s)
 	}
 
+	if userns.RunningInUserNS() {
```

Comment:
Perhaps it's worth adding a comment here to describe why we need to do this additional work for rootless

Reply:
added